static/js/typer.js

```diff
@@ -0,0 +1,141 @@
+// Copied from this project:
+// https://steven.codes/typerjs/docs/index.html
+
+var Typer = function (element) {
```

Comment:
Would you be able to make it so the `sudo` doesn't delete as it's the same in both commands? we have an example in this repo doing the same thing https://github.com/canonical-web-and-design/jaas.ai/blob/master/static/js/cli-typer.js (This has more options in the array but you could pretty much copy it). Here it is on [live](https://jaas.ai/)

Reply:
Would it be okay just to take the `$ sudo` out of the `span`? The`$ sudo` stays now!